odoo/addons/base/models/ir_model.py

```diff
@@ -1691,6 +1691,11 @@ class IrModelAccess(models.Model):
     perm_create = fields.Boolean(string='Create Access')
     perm_unlink = fields.Boolean(string='Delete Access')
 
+    @api.depends('model_id', 'group_id')
+    def _compute_name(self):
+        for acl in self:
+            acl.name = '%s access %s' % (acl.model_id.name, acl.group_id.name or 'public')
```

Comment:
`model_id.name` is translatable, so you are probably missing a `@api.depends_context('lang')`

Reply:
I want the English name, because acl.name is not translatable, so I'll fix the context